README.md

```diff
@@ -21,23 +21,24 @@ ROS messages and services used across many ROS packages in the project.
 | `batteries`            | Batteries      | CAN Transceiver          | Local/Remote Transceiver                    |
 | `desired_heading`      | DesiredHeading | Local Pathfinding        | Controller, Boat Simulator                  |
 | `data_sensors`         | GenericSensors | CAN Transceiver          | Local/Remote Transceiver                    |
-| `global_path`          | GlobalPath     | Local/Remote Transceiver | Local Pathfinding                           |
+| `global_path`          | Path           | Local/Remote Transceiver | Local Pathfinding                           |
 | `gps`                  | GPS            | CAN Transceiver          | Local/Remote Transceiver, Local Pathfinding |
+| `local_path_data`      | LPathData      | Local Pathfinding        | Local/Remote Transceiver                    |
 | `mock_gps`             | GPS            | Boat Simulator           | CAN Transceiver                             |
 | `filtered_wind_sensor` | WindSensor     | CAN Transceiver          | Local/Remote Transceiver, Local Pathfinding |
 | `mock_wind_sensors`    | WindSensors    | Boat Simulator           | CAN Transceiver                             |
 | `wind_sensors`         | WindSensors    | CAN Transceiver          | Local/Remote Transceiver                    |
 
 ### Project-wide Internal Interfaces
 
-| Interface           | Used In                            |
-| ------------------- | ---------------------------------- |
-| HelperAISShip       | AISShips                           |
-| HelperBattery       | Batteries                          |
-| HelperGenericSensor | GenericSensors                     |
-| HelperHeading       | DesiredHeading, GPS, HelperAISShip |
-| HelperLatLon        | GlobalPath, GPS, HelperAISShip     |
-| HelperSpeed         | GPS, HelperAISShip, WindSensor     |
+| Interface           | Used In                                   |
+| ------------------- | ----------------------------------------- |
+| HelperAISShip       | AISShips                                  |
+| HelperBattery       | Batteries                                 |
+| HelperGenericSensor | GenericSensors                            |
+| HelperHeading       | DesiredHeading, GPS, HelperAISShip        |
+| HelperLatLon        | GlobalPath, GPS, LocalPath, HelperAISShip |
```

Comment:
- Rename GlobalPath to Path
- Remove LocalPath

Reply:
Done! Sorry I missed these